### ISSUE TITLE
I use matconvnet to reproduce your work,and I have some questions~

### ISSUE THREAD

**Issue by icyzhang0923** [rohitgirdhar/ActionVLAD#15]
I have some questions about the action vlad:
1. How to sample the 25 frames from each video, random or uniform?
2. How to preprocess the frames in Tensorflow?
3. How to combine the two stream? use SVM classifier?
4. I'm not sure the clip gradients at 5 l2 norm, to calculate which gradients' l2 norm，accumalated gradients,or the average gradient over the minibatch?

**Comment by rohitgirdhar** [comment]
1. We found nearly similar performance either way
2. Please refer to `preprocessing/vgg_ucf_preprocessing.py` for details
3. We used weighted averaging of the softmax probabilities (similar to previous methods)
4. [This](https://github.com/rohitgirdhar/ActionVLAD/blob/94e51bc5d07ee6aad935743bfe99deb1a1d40530/train_image_classifier.py#L785) is how we use the `clip_gradients`
